Replace disconnect button with disband button always

taywrobel opened the pull request:
fixes #184

taywrobel committed:
Replace Disconnect Button with Disband

Now always show Disband as the disconnect
or disband functionality.  It makes more
sense than having it change if there are
guests or not.

dgreenhalgh commented:
@ejohnson44, could you take a look at this one?

reidbaker commented:
This looks a lot bigger than just this change. Which request is the base of the rest of this stuff.

taywrobel committed:
Change disband button to be statically set once

Only sets the disband button's name and functionality
once, since it no longer needs to toggle between
disband and disconnect

taywrobel committed:
Merge branch 'master' into disband
